client/components/EntryFormRow/index.js

```diff
@@ -1,20 +1,57 @@
 import React, { Component, PropTypes } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { getCategories } from 'modules/categories';
+import { actions as AppActions } from 'modules/transactions';
 import DataSelector from './DataSelector';
 import styles from './style.scss';
 
+@connect(
+  state => ({
+    categories: getCategories(state)
+  }),
+  (dispatch => ({
+    actions: bindActionCreators(AppActions, dispatch)
+  }))
+)
 class EntryFormRow extends Component {
   static propTypes = {
-    categories: PropTypes.object.isRequired
+    categories: PropTypes.object.isRequired,
+    actions: PropTypes.object.isRequired
   }
 
   state = {
     categoryId: '16',
```

Comment:
I know this wasn't part of your PR but I'm wondering if we could benefit from having a getDefaultCategory selector/helper/etc here instead of putting the magic number in the view.